app/i18n/messages/sv.json

```diff
@@ -48,7 +48,8 @@
   "common.totalPriceLabel": "Total pris",
   "common.priceWithVAT": "{price}€ (inkl. moms {vat}%)",
   "payment.title": "Betalning avbruten",
-  "payment.text": "Prova igen om 15 minuter",
+  "payment.text": "Betalning avbruten.",
+  "payment.return": "Lämna tillbaka",
```

Comment:
Where did you get this translation from?

Reply:
From @SamuelSaarentola  https://helsinkisolutionoffice.atlassian.net/browse/VAR-179

Reply:
@SamuelSaarentola I'm not a native Swedish speaker, so I'm not going to argue about this, I just wanted to mention that I read "lämnä tillbaka" as "return" in the same way as you'll return a book to the library. Just FYI :-)

Reply:
@badrange @Nizar-Rahme Im not either, more correct would be just **tillbaka**